src/azul/lambdas.py

```diff
@@ -97,11 +97,18 @@ class Lambdas:
     def _lambda(self):
         return aws.lambda_
 
-    def list_lambdas(self) -> list[Lambda]:
+    def list_lambdas(self, deployment_only: bool = False) -> list[Lambda]:
```

Comment:
```suggestion
    def list_lambdas(self, deployment: str | None = None) -> list[Lambda]:
```

where `None` means any deployment.